pkg/workflows/create.go

```diff
@@ -247,13 +249,30 @@ func (s *MoveClusterManagementTask) Run(ctx context.Context, commandContext *tas
 		return &CollectDiagnosticsTask{}
 	}
 
-	return &InstallEksaComponentsTask{}
+	return &InstallEksdComponentsTask{}
 }
 
 func (s *MoveClusterManagementTask) Name() string {
 	return "capi-management-move"
 }
 
+// InstallEksdComponentsTask implementation
+
+func (s *InstallEksdComponentsTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	if !commandContext.BootstrapCluster.ExistingManagement {
```

Comment:
this is problematic

What happens if this is a workload cluster but we are installing a new eksd version?